doc_test.go

```diff
@@ -0,0 +1,35 @@
+package trie
+
+import "fmt"
+
+func Example() {
```

Comment:
https://pkg.go.dev/github.com/fluhus/godoc-tricks#Examples

```suggestion
func ExampleSearchAll() {
```